cardano-db-sync/src/Cardano/DbSync/Sync.hs

```diff
@@ -334,13 +336,15 @@ dbSyncProtocols syncEnv metricsSetters _version codecs _connectionId =
                             (runDbThread syncEnv metricsSetters actionQueue)
                             (runOfflineFetchThread syncEnv)
                         )
-                        ( runPipelinedPeer
-                            localChainSyncTracer
-                            (cChainSyncCodec codecs)
-                            channel
-                            ( chainSyncClientPeerPipelined $
-                                chainSyncClient metricsSetters tracer (fst <$> latestPoints) currentTip actionQueue
+                        ( race_
```

Comment:
Another inquisitive question, how come we race here? which ever finishes first would cancel the other but not grasping which is likely to win/lose?

Reply:
They both should run forever, so we don't care which wins of loses.

Reply:
ah cool, so it's just about being able to do them concurrently 👍